lib/src/widget/asset_picker_viewer.dart

```diff
@@ -62,24 +62,24 @@ class AssetPickerViewer extends StatefulWidget {
   /// Static method to push with navigator.
   /// 跳转至选择预览的静态方法
   static Future<List<AssetEntity>> pushToViewer(
-    BuildContext context, {
-    int currentIndex = 0,
-    @required List<AssetEntity> assets,
-    @required ThemeData themeData,
-    List<AssetEntity> selectedAssets,
-    AssetPickerProvider selectorProvider,
-    TextDelegate textDelegate,
-  }) async {
+      BuildContext context, {
```

Comment:
Please prevent any format commit next time, because `flutter format` was preferred in here rather than `dartfmt`.

Reply:
OK, sorry about that. I'm new to Flutter development.